sim/lib/stdlib.c

```diff
@@ -19,3 +19,17 @@ void exit(int status) {
 
   __builtin_unreachable();
 }
+
+unsigned long clock() {
+  // reading first byte latches whole 32-bit value
+  // so it needs to be read byte by byte in correct order
+  unsigned long ticks;
+  for(int i=0; i < 4; i++) {
+    ((unsigned char *)&ticks)[i] = *((volatile unsigned char *)0xFFF0 + i);
```

Comment:
nit: just char, here and throughout. I made char unsigned for efficiency, so while unsigned char means the same thing, I don't want people to go spelunking into the SDK for "the best way to write llvm-mos C" and come away with the idea that they have to put unsigned char everywhere.